src/legacy/core_plugins/metrics/public/kbn_vis_types/index.js

```diff
@@ -77,7 +77,7 @@ export default function MetricsVisProvider(Private, i18n) {
       component: require('../components/vis_editor')
     },
     options: {
-      showQueryBar: false,
+      showQueryBar: true,
       showFilterBar: false,
```

Comment:
Is there any reason we wouldn't want to show the filter bar? Aren't they being applied too?

Reply:
:( oh lukas you are the missing bars guy 😬 
So this comes from this PR: https://github.com/elastic/kibana/pull/12770
That is closing this ticket 2 years ago: https://github.com/elastic/kibana/issues/12761

Reply:
Let me check with Peter to understand things better

Reply:
ok @lukasolson I prefer not to switch on the filterBar because it can be a bit confusing when you are editing a TSVB viz on an index-pattern different from the default one: the FilterBar will shows the options on the default one, that cannot always be applied to the used index-pattern.
I prefer to postpone this when we are making explicit the index-pattern on filterbar and on querybar